### PR TITLE
aria2, aria2c: add French translation & page edit

### PR DESCRIPTION
**Pull request by Nico385412**
- **Version of the command being documented (if known):**  

- [x] The page(s) are in the correct platform directories: `common`, `linux`, `osx`, `windows`, `sunos`, `android`, etc.
- [x] The page(s) have at most 8 examples.
- [ ] The page description(s) have links to documentation or a homepage.
- [ ] The page(s) follow the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [ ] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message).

Hi 👋 
A new day a new translation, and also a page edit.

I noticed the english page has an error in his documentation with the `--force-sequential`

According to the documentation it performs a sequential download, not a parallel download as tldr page says.
I suggest to fix this :p 

image

### CODE REVIEW

**Comment by Nico385412** [comment]
> This flag description is a little confusing, because of the `Default: false`. Does that mean that `--force-sequential true` sets the option to true, but `--force-sequential` sets it to false?

I aggree it's realy confusing 😅. I'll figure it out this afternoon

**Comment by Nico385412** [comment]
Ok then, after a lot of trouble to install it i figure it out
`--force-sequential` without true/false value **download in parallel** 
However if you set the value to true, it will download in sequential.

By default aria2c download in parallel without the option. you dont need the option `--force-sequential` **as long as the urls point to the same file**

However if the files are =/= you can set `--force-sequential` without parameter do achieve it. (sounds more like a trick ...)

So i have to rollback my changes, the en page is correct ^^"

**Comment by marchersimon** [comment]
The example is still very confusing. How about we change it to:
```markdown
- Download multiple (different) files in parallel:

`aria2c --force-sequential {{false}} {{url_1}} {{url_2}}`

```

**Comment by Nico385412** [comment]
> The example is still very confusing. How about we change it to:
> 
> ```gfm
> - Download multiple (different) files in parallel:
> 
> `aria2c --force-sequential {{false}} {{url_1}} {{url_2}}`
> ```

Yes definitely ! i don't what the others thinks ?

**Review by EmilySeville7cfg — APPROVED** [review]

**Comment by marchersimon** [comment]
@Nico385412 I think you can apply the change

**Comment by Nico385412** [comment]
> @Nico385412 I think you can apply the change

Done ! @marchersimon

File: pages.fr/common/aria2c.md [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+
+`aria2c {{url}}`
+
+- Télécharge un fichier via l'url spécifié en choisissant le nom de ce dernier :
+
+`aria2c --out={{nom_de_fichier}} {{url}}`
+
+- Télécharge plusieurs fichiers (différents) en parallèle :
+
+`aria2c --force-sequential {{url_1}} {{url_2}}`
```
Comment (marchersimon) [inline pages.fr/common/aria2c.md:17]:
```suggestion
`aria2c --force-sequential {{false}} {{url_1}} {{url_2}}`
```

File: pages/common/aria2c.md [hunk 2]
```diff
@@ -12,7 +12,7 @@
 
 `aria2c --out={{filename}} {{url}}`
 
-- Download multiple files in parallel:
+- Download multiple (different) files in parallel:
 
 `aria2c --force-sequential {{url_1}} {{url_2}}`
```
Comment (marchersimon) [inline pages/common/aria2c.md:17]:
```suggestion
`aria2c --force-sequential {{false}} {{url_1}} {{url_2}}`
```

**Review by marchersimon — APPROVED** [review]
Thanks!

**Pull request merged**